js/libs/builder.js

```diff
@@ -5,9 +5,17 @@ var data = Fliplet.Widget.getData(widgetId) || {};
 var isFormInSlider = false;
 
 Fliplet.Widget.findParents({ instanceId: widgetId }).then(function(parents) {
-  data.slideId = parents.length && parents[0].slideId;
-  data.sliderContainerId = parents.length > 1 && parents[1].sliderId;
-  isFormInSlider = parents.length > 1 && parents[1].sliderId;
+  const formSlideParent = parents.find(parent =>
+    parent.package === 'com.fliplet.slide' || parent.name === 'Slide'
+  );
+
+  const formSliderParent = parents.find(parent =>
+    parent.package === 'com.fliplet.slider-container' || parent.name === 'Slider container'
+  );
+
+  data.slideId = formSlideParent.length && formSlideParent.slideId;
```

Comment:
@armine-fliplet Is this correct? You seem to be accessing `slideId` from `formSlideParent` which is an array.